servers/src/main/java/tachyon/master/MasterSource.java

```diff
@@ -38,22 +38,58 @@
   private static final String MASTER_SOURCE_NAME = "master";
   private boolean mGaugesRegistered = false;
   private final MetricRegistry mMetricRegistry = new MetricRegistry();
+
+  private final Counter mCompleteFileOps =
```

Comment:
ideally these definitions are divided into the same logical blocks as the documentation